build/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:3.11.0a3-bullseye
-COPY ./python /python
-WORKDIR /python
-RUN python ./tester.py
+# FROM sgrun14/python-pip-speedtest
+FROM python:latest
+RUN pip install speedtest-cli
+WORKDIR /usr/src/app
+COPY ./python .
+ENTRYPOINT ["python", "./tester.py"]
```

Comment:
Using `ENTRYPOINT` instead of `RUN`
This runs when the container starts instead of during the image build process